src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static removeFromAlive(animal) {
+    const index = Animal.alive.indexOf(animal);
+    if (index !== -1) {
+      Animal.alive.splice(index, 1);
+    }
+  }
 }
 
```

Comment:
https://github.com/mate-academy/js_herbivores_and_carnivores/issues/952
That's why you have failed tests

```suggestion
Animal.alive = [];
```